tests/UntrustedPSPIntegrationTests.js

```diff
@@ -26,19 +28,188 @@ gpii.loadTestingSupport();
+// The expected preferences change after running each test sequence. The array indexes maps the array indexes of test sequences.
+gpii.test.untrusted.pspIntegration.expectedPrefsChange = [
+    // 0
+    {
+        "flat": {
+            "contexts": {
+                "gpii-default": {
+                    "name": "Default preferences",
+                    "preferences": {
+                        "http://registry.gpii.net/common/magnification": 3
+                    }
+                }
+            }
+        }
+    },
+    // 1
+    {
+        "flat": {
+            "contexts": {
+                "gpii-default": {
+                    "name": "Default preferences",
+                    "preferences": {
+                        "http://registry.gpii.net/applications/org.gnome.desktop.a11y.magnifier": {
+                            "http://registry.gpii.net/common/magnification": 3
+                        }
+                    }
+                }
+            }
+        }
+    },
+    // 2
+    {
+        "flat": {
+            "contexts": {
+                "gpii-default": {
+                    "name": "Default preferences",
+                    "preferences": {
+                        "http://registry.gpii.net/common/magnification": 3,
+                        "http://registry.gpii.net/common/volume": 0.75
+                    }
+                }
+            }
+        }
+    },
+    // 3
+    {
+    },
+    // 4
+    {
+        "flat": {
+            "contexts": {
+                "gpii-default": {
+                    "name": "Default preferences",
+                    "preferences": {
+                        "http://registry.gpii.net/common/magnification": 3
+                    }
+                }
+            }
+        }
+    },
+    // 5
+    {
+        "flat": {
+            "contexts": {
+                "gpii-default": {
+                    "name": "Default preferences",
+                    "preferences": {
+                        "http://registry.gpii.net/common/magnification": 3
+                    }
+                }
+            }
+        }
+    }
+];
+
+// The sequence to be added to the start of each test sequence to verify the initial raw preferences
+gpii.test.untrusted.pspIntegration.startSequence = fluid.freezeRecursive([
+    {
+        func: "{rawPrefsAtStart}.send"
+    }, {
+        event: "{rawPrefsAtStart}.events.onComplete",
+        listener: "gpii.test.untrusted.pspIntegration.verifyRawPrefsAtStart",
+        args: ["{that}", "{arguments}.0"]
+    }
+]);
+
+// The sequence to be appended to the end of each test sequence to verify the updated raw preferences
+gpii.test.untrusted.pspIntegration.endSequence = [
+    {
+        func: "{rawPrefsAtEnd}.send"
+    }, {
+        event: "{rawPrefsAtEnd}.events.onComplete",
+        listener: "gpii.test.untrusted.pspIntegration.verifyRawPrefsAtEnd"
+        // "i" needs to be dynamically provided at constructing testDefs with the current sequence number to
```

Comment:
Such a shame we never had time to start working with the FLUID-5903 grade sequence system